code/modules/shuttle/on_move.dm

```diff
@@ -16,13 +16,13 @@
     // Stationary ports shouldn't move, mobile ones move themselves
     return 0
 
-/obj/machinery/door/onShuttleMove()
+/obj/machinery/door/airlock/onShuttleMove()
```

Comment:
Think it makes sense to be just airlocks. There aren't any firedoors on shuttles that I'm aware of... and wooden doors automagically closing is goofy.